### DISCUSSION TITLE
Adventure map, "Month of the Creature" and "Week of the Creature" windows [Done]

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#3310] [Ideas]
For these windows, I suggest to use the current first sentence as a title in yellow with an exclamation mark instead of a period at the end of the sentence: "Astrologers proclaim Month of the Ogre!" and "Astrologers proclaim Week of the Master Swordsman!".

**Edit 2021/12/22:** After some thought, I think the titles "Month of the Creature!" and "Week of the Creature!" would be more appropriate. :)

![Month of Ogre](https://user-images.githubusercontent.com/43583994/116582514-c043ac80-a915-11eb-93ae-a66f9566d164.png)

![Week of Master Swordsman](https://user-images.githubusercontent.com/43583994/116582545-c76aba80-a915-11eb-8ebe-b18eb0b6a054.png)

I suggest also adding the creature's image with a frame between the end of the text and the "OKAY" button.

In the example, with the Orc Chief:

![Creature with frame fh2 coupé 01](https://user-images.githubusercontent.com/43583994/116582578-cf2a5f00-a915-11eb-90a6-aacb393cd476.png)

**Comment by ihhub** [comment]
This is a good suggestion. @Branikolog , what do you think?

**Reply by Branikolog** [reply]
Agreed.
Making yellow title is what we want to add to the majority of messages in game.
Regarding the icon of creature: I think, the idea is good, but I'm afraid, the window in this case can look a bit overloaded.

**Reply by LeHerosInconnu** [reply]
Hello @ihhub and @Branikolog,

> Regarding the icon of creature: I think, the idea is good, but I'm afraid, the window in this case can look a bit overloaded.

It's okay, it will be equivalent as the Hill Fort with two creatures.

In fheroes2:

![Hill Fort fh2 01](https://user-images.githubusercontent.com/43583994/118137978-192b3e80-b406-11eb-9110-efa95674787f.png)
![Hill Fort fh2 02](https://user-images.githubusercontent.com/43583994/118137992-1cbec580-b406-11eb-8e60-50bcda25d50b.png)

**Comment by LeHerosInconnu** [comment]
Other titles have been implemented.
The creature graphic has not been added.
I close the discussion.
